contrib/test_jenkins.sh

```diff
@@ -659,6 +659,25 @@ check_make_distcheck() {
 	fi
 }
 
+check_config_h() {
+	echo 1..1 > check_config_h.tap
+
+	srcdir=$PWD/../src
+
+	# Check if all .c files include config.h
+	echo "==== Checking for config.h files in directory $srcdir ===="
+
+	missing=`find $srcdir -name \*.c | xargs grep -LP '\#\s*include\s+"config.h"'`
```

Comment:
ok, but can use simple grep, like `grep '\#include[ ]*\"config\.h\"'`

Reply:
what about
```
#ifdef HAVE_CONFIG_H
#  include "config.h"
#endif
```

Reply:
would  need to add `[ ]*` after `#` as well
never mind, you exp LGTM

Reply:
what about .cc files like in ucx_perftest?
also, do we expect this include from header files?
or SRC files must include config.h before including any other header file?

Reply:
> what about .cc files like in ucx_perftest?

hmmm, good question... actually perftest should not use internal UCX headers, but it uses. I can add evaluation for such files, but not sure it is correct

> also, do we expect this include from header files?
> or SRC files must include config.h before including any other header file?

source file (not header) must start with ```#include "config.h"```, internal include files - not mandatory (may/may not - both acceptable, imho)

Reply:
> hmmm, good question... actually perftest should not use internal UCX headers, but it uses. I can add evaluation for such files, but not sure it is correct

I think it's ok to use config.h

> source file (not header) must start with #include "config.h", internal include files - not mandatory (may/may not - both acceptable, imho)

Can we check that config.h is included first ?

Reply:
> I think it's ok to use config.h

ok, will update test

> Can we check that config.h is included first ?
the problem is not all include files should be started with config.h, for example external headers should not include it, that is why .c files should be started with config.h